grafana/perf/regression-summary.json

```diff
@@ -70,20 +71,284 @@
       "type": "text"
     },
     {
-      "datasource": {
-        "type": "elasticsearch",
-        "uid": "eh85eR3nz"
-      },
+      "description": "OVN - 09/19",
       "fieldConfig": {
         "defaults": {
           "color": {
-            "mode": "thresholds"
+            "fixedColor": "super-light-blue",
+            "mode": "fixed"
           },
           "custom": {
-            "align": "auto",
-            "displayMode": "auto"
+            "fillOpacity": 70,
+            "lineWidth": 3
           },
-          "mappings": [],
+          "mappings": [
+            {
+              "options": {
+                "102": {
+                  "index": 13,
+                  "text": "1.0.2"
+                },
+                "110": {
+                  "index": 14,
+                  "text": "1.1.0"
+                },
+                "120": {
+                  "index": 15,
+                  "text": "1.2.0"
+                },
+                "121": {
+                  "index": 16,
+                  "text": "1.2.1"
+                },
+                "130": {
+                  "index": 45,
+                  "text": "1.3.0"
+                },
```

Comment:
Kata operator version, I assume?  So unfortunate that JSON does not allow comments.

Reply:
No, the plan is to shift to [jsonet](https://jsonnet.org/)

Reply:
That was just a comment that those look like Kata operator versions.  No change is actually needed here.